mappings/net/minecraft/world/gen/feature/GlowLichenFeatureConfig.mapping

```diff
@@ -1,20 +1,20 @@
 CLASS net/minecraft/class_5780 net/minecraft/world/gen/feature/GlowLichenFeatureConfig
 	FIELD field_28429 CODEC Lcom/mojang/serialization/Codec;
 	FIELD field_28430 searchRange I
-	FIELD field_28431 placeOnFloor Z
-	FIELD field_28432 placeOnCeiling Z
-	FIELD field_28433 placeOnWalls Z
-	FIELD field_28434 spreadChance F
-	FIELD field_28435 canPlaceOn Ljava/util/List;
+	FIELD field_28431 canPlaceOnFloor Z
+	FIELD field_28432 canPlaceOnCeiling Z
+	FIELD field_28433 canPlaceOnWall Z
+	FIELD field_28434 chanceOfSpreading F
+	FIELD field_28435 canBePlacedOn Ljava/util/List;
 	FIELD field_28436 directions Ljava/util/List;
 	METHOD <init> (IZZZFLjava/util/List;)V
 		ARG 1 searchRange
-		ARG 2 placeOnFloor
-		ARG 3 placeOnCeiling
-		ARG 4 placeOnWalls
-		ARG 5 spreadChance
-		ARG 6 canPlaceOn
-	METHOD method_33396 canGrowOn (Lnet/minecraft/class_2248;)Z
+		ARG 2 canPlaceOnFloor
+		ARG 3 canPlaceOnCeiling
+		ARG 4 canPlaceOnWall
+		ARG 5 chanceOfSpreading
+		ARG 6 canBePlacedOn
+	METHOD method_33396 canBePlacedOn (Lnet/minecraft/class_2248;)Z
```

Comment:
I personally like the old names better

Reply:
These names are directly from the codec, but alright I'll change them back